www/profile/inner.js

```diff
@@ -463,6 +498,14 @@ define([
         editor.setOption('readOnly', false);
         cm.configureTheme(common, function () {});
 
+        editor.on("keydown", function (cm, event) {
```

Comment:
CodeMirror directly includes a way to handle keyboard events:
```javascript
editor.setOption("extraKeys", {
    "Esc": function () {
        cm.getInputField().blur();
        $(save).focus();
    }
}
```
This would avoid any conflict with other shortcuts that can be added.